tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/Npgsql/NpgsqlConstants.cs

```diff
@@ -3,11 +3,13 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using Datadog.Trace.Configuration;
+
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.Npgsql
 {
     internal static class NpgsqlConstants
     {
-        public const string SqlCommandIntegrationName = "NpgsqlCommand";
+        public const string SqlCommandIntegrationName = nameof(IntegrationIds.AdoNet);
```

Comment:
Was there a reason for changing all these integration names? 🤔 Are they actually used for anything besides information in logs?

Reply:
I changed them to match with the `IntegrationIds` enum. This affects the both `DD_DISABLED_INTEGRATIONS` and `DD_TRACE_<INTEGRATION_NAME>_ENABLED` environment variable.

The behavior in callsite is having all `ADO.NET` integrations grouped, that way we can disable them very easy. 

In calltarget we weren't following that behavior, so disabling `ADO.NET` integrations is a mess right now, because to disable 1 provider, you have to disable the integrations with the ProviderName and the integrations from the base class with the `ADO.NET` and I think this changes of behavior doesn’t make sense.